packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/TableSingleRowSelection.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useCallback } from 'react';
+import Table from 'terra-table';
+
+const tableDataJSON = {
+  cols: [
+    { id: 'Column-0', displayName: 'Patient' },
+    { id: 'Column-1', displayName: 'Location' },
+    { id: 'Column-2', displayName: 'Illness Severity' },
+  ],
+  rows: [
+    {
+      id: '1',
+      isSelectable: true,
+      cells: [
+        { content: 'Fleck, Arthur' },
+        { content: '1007-MTN' },
+        { content: 'Unstable' },
+      ],
+    },
+    {
+      id: '2',
+      isSelectable: true,
+      cells: [
+        { content: 'Wayne, Bruce' },
+        { content: '1107-MTN-DR' },
+        { content: 'Stable' },
+
+      ],
+    },
+    {
+      id: '3',
+      isSelectable: true,
+      cells: [
+        { content: 'Carr, Alicia' },
+        { content: '1007-MTN' },
+        { content: 'Unstable' },
+
+      ],
+    },
+    {
+      id: '4',
+      isSelectable: true,
+      cells: [
+        { content: 'McMurphy, Donald' },
+        { content: '1024-MTN', isMasked: true },
+        { content: 'Stable' },
+      ],
+    },
+    {
+      id: '5',
+      isSelectable: true,
+      cells: [
+        { content: 'Peters, Timothy' },
+        { content: '1207-MTN' },
+        { content: 'Unstable', isMasked: true },
+      ],
+    },
+  ],
+};
+
+const TableSingleRowSelection = () => {
+  const rowHeaderIndex = 0;
+  const { cols, rows } = tableDataJSON;
+  const [rowData, setRowData] = useState(rows);
+
+  const onRowSelect = useCallback((rowId) => {
```

Comment:
In this example would we want to keep each row selected until it is deselected, or clear the previous row when a new one is selected? Currently we allow for both depending on the callback implementation, but the use-case of selecting multiple rows in single-select mode doesn't really make much sense when multi-select mode is available.

Reply:
Let me check.  Maybe I missed updating a test file

Reply:
@cm9361 I still see the issue with multiple rows being selectable in the test file.

Reply:
Works with the latest deployment.